gamemodes/terrortown/gamemode/shared/hud_elements/tttinfopanel/pure_skin_playerinfo.lua

```diff
@@ -31,6 +31,8 @@ if CLIENT then
 		minsize = {w = 240, h = 146}
 	}
 
+	local healthPulsate = CreateConVar("ttt2_hud_pulsate_health", "1", FCVAR_ARCHIVE)
```

Comment:
This should be defined somewhere outside the Hudelement, and since this is a generic feature not directly tied to the design, we might want to also add this to the old HUD).

Reply:
I already thought that this would come up :)
I defined it in the hud element as I really had no freakin idea where I should put it else.

So.. where should I put it?
Also, not in misc but under a different menu? I mean sure possible, I just felt it would fit more under misc.

Also, no I'm actually strictly against adding this to the old hud. The old hud is, imo, legacy and should not be altered in such a way.

Reply:
I'm pretty sure you can remnove the convar and add the setting here, then it will appear in the HUD switcher:

https://github.com/TTT-2/TTT2/blob/42f5bb613bcdeb948dd19ea3b0370b5ec18d45c9/gamemodes/terrortown/gamemode/shared/huds/base_huds/scalable_hud/cl_init.lua#L32-L46

Altough you should probably only do this in the cl_init of the pure_skin hud.